Plugin~/WebRTCPlugin/Codec/NvCodec/NvEncoderImpl.cpp

```diff
@@ -105,6 +109,13 @@ namespace webrtc
             return WEBRTC_VIDEO_CODEC_ERROR;
         }
 
+        m_configurations[0].sending = false;
+        m_configurations[0].max_frame_rate = static_cast<float>(m_codec.maxFramerate);
+        m_configurations[0].frame_dropping_on = m_codec.H264()->frameDroppingOn;
```

Comment:
I observe frame dropping when the bitrate of the actual encoded picture exceeds the network bandwidth available (i.e. libwebrtc has failed to manage the bitrate via SetRates, or it had the wrong network bandwidth).

Reply:
Thanks for sharing.
The issue https://github.com/Unity-Technologies/com.unity.webrtc/issues/703 should be fixed by the PR, and I will fix the issue of frame dropping on another occasion.